elementary/monitor/alerts/test.py

```diff
@@ -150,45 +126,64 @@ def _get_test_description(self):
             return None
 
     def to_slack(self, is_slack_workflow: bool = False) -> SlackMessageSchema:
-        alert_fields = self.get_alert_fields()
         icon = self.slack_message_builder.get_slack_status_icon(self.status)
 
         if is_slack_workflow:
             return SlackMessageSchema(text=json.dumps(self.__dict__))
 
         title = [
-            self.slack_message_builder.create_header_block(f"{icon} dbt test alert"),
-            self.slack_message_builder.create_context_block(
-                [
-                    f"*Test:* {self.test_short_name if self.test_short_name else self.test_name}     |",
-                    f"*Status:* {self.status}     |",
-                    f"*{self.detected_at.strftime(DATETIME_FORMAT)}*",
-                ],
-            ),
+            self.slack_message_builder.create_header_block(f"{icon} dbt test alert")
         ]
+        if self.alert_suppression_interval:
+            title.extend(
+                [
+                    self.slack_message_builder.create_context_block(
+                        [
+                            f"*Test:* {self.test_short_name if self.test_short_name else self.test_name} - {self.test_sub_type_display_name}     |",
```

Comment:
How can a test not have a short name?
According to `{% set test_short_name = test_metadata.get('name') %}`.

Reply:
A singular test has no short name 🤷🏻